src/Uno.UI/UI/Xaml/Controls/ItemsControl/ItemsControl.cs

```diff
@@ -19,18 +19,19 @@
 
 #if __ANDROID__
 using Android.Graphics;
-
 using View = Android.Views.View;
+using ViewGroup = Android.Views.ViewGroup;
```

Comment:
let's use `NativeView` instead.

Reply:
I find `NativeView` confusing as we already have a `View`, whereas a `ViewGroup` is a `View` that can have children (`ViewGroup`s and `View`s are the same thing except on Android AFAIK). I've added an underscore (`_ViewGroup`) to make it extra clear that it's just an alias. This is already used elsewhere in the codebase.